.github/workflows/release.yaml

```diff
@@ -0,0 +1,182 @@
+#
+# Copyright Red Hat
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+name: Release
+
+on:
+  push:
+    tags:
+      - 'v[0-9]+.[0-9]+.[0-9]+'
+      - 'v[0-9]+.[0-9]+.[0-9]+-rc.[0-9]+'
+
+env:
+  OPERATOR_SDK_DL_URL: https://github.com/operator-framework/operator-sdk/releases/download/v1.28.0
+  OPERATOR_SDK_CLI: ./operator-sdk
+
+jobs:
+  version-check:
+    runs-on: ubuntu-latest
+    env:
+      CACHED_CSV_VERSION: .cache/csv_version.txt
+      CACHED_CSV_CONTAINER_IMAGE_TAG: .cache/csv_container_image_tag.txt
+      CACHED_CSV_NAME_TAG: .cache/csv_name_tag.txt
+      CACHED_BUNDLE_VERSION: .cache/bundle_version.txt
+      CACHED_BUNDLE_CONTAINER_IMAGE_TAG: .cache/bundle_container_image_tag.txt
+      CACHED_BUNDLE_NAME_TAG: .cache/bundle_name_tag.txt
+      CACHED_MANAGER_IMAGE_TAG: .cache/manager_image_tag.txt
+    steps:
+    - name: Checkout registry-operator source code
+      uses: actions/checkout@v2
```

Comment:
can we pin all actions to their commit SHA?

Reply:
addressed in https://github.com/devfile/registry-operator/pull/56/commits/712c0887145e86653950c61944c22673c32116f1